css/main.css

```diff
@@ -47,6 +48,33 @@ h3 {
   max-width: 400px;
 }
 
+.boxed {
+  border: 1px solid black ;
+  max-width: 70%;
+  text-align: center;
+  color: #333;
+  font-family: 'Open Sans', sans-serif;
+  /*font-size: 14px; */
+  -webkit-font-smoothing: antialiased; 
+  padding: 0 5% 0 5%; 
+  position: relative; 
+  width: 60%;
+  left: 15%;
+  right: 15%
+  
+}
+
+.col-wrapper{width:100%; 
```

Comment:
nit: maybe put a nl after `{` and before `}` to keep format consistent and clear